public/assets/css/style.css

```diff
@@ -34,18 +34,17 @@ body {
     align-items: center;
 }
 .button {
-    color: white;
     text-align: center;
     padding: 30px;
     margin-top: 2rem;
     background-color: #3b3b3b;
     border-color: #413e3e;
     border-radius: 100% 100% 100% 100%;
-    width: 17rem;
-    font-size: 2rem;
+    width: 12rem;
```

Comment:
prefer padding to handle button size: if you change button content, the fixed width may break the design